source/locale/pt_BR/LC_MESSAGES/architecture.po

```diff
@@ -458,31 +457,31 @@ msgstr ""
 "criação, visualização, edição e remoção de fluxos de processamento de "
 "dados."
 
-#: ../../source/architecture.rst:192
+#: ../../source/architecture.rst:193
 msgid ""
 "**Notifications**: View system notifications (unified real time and "
 "history)"
 msgstr ""
 "**Notificações**: visualiza as notificações do sistema (em tempo real e "
 "histórico unificados)"
 
-#: ../../source/architecture.rst:195
+#: ../../source/architecture.rst:196
 msgid "Image manager"
 msgstr "Image manager"
 
-#: ../../source/architecture.rst:197
+#: ../../source/architecture.rst:198
 msgid ""
 "This component is responsible for device (firmware) image storage and "
 "retrieval. It is used by the firmware update mechanism."
 msgstr ""
-"Este componente é responsável pelo armazenamento e recuperação de imagens "
-"de firmware de dispositivos."
+"Este componente é responsável pelo armazenamento e recuperação de imagens"
+" de firmware de dispositivos."
```

Comment:
It seems that it missed the second phrase's translation.
`Ele é utilizado pelo mecanismo de atualização de *firmware*`